lib/Auth.js

```diff
@@ -1,22 +1,29 @@
 import auth0 from 'auth0-js'
+import Router from 'next/router'
 
 class Auth {
   auth0 = new auth0.WebAuth({
-    domain: '<Your Domain>',
-    clientID: '<Your client ID>',
+    domain: process.env.AUTH0_DOMAIN,
+    clientID: process.env.AUTH0_CLIENT_ID,
     redirectUri: 'http://localhost:3000/callback',
```

Comment:
The hostname for the redirect URI will need to use the environment variable for the host